iphone/Maps/Classes/ClipboardMonitor.mm

```diff
@@ -0,0 +1,116 @@
+#import "MapsAppDelegate.h"
+#import "ClipboardMonitor.h"
+#import "UIKitCategories.h"
+#import "MapsAppDelegate.h"
+#import <CoreApi/DeepLinkParser.h>
```

Comment:
1. Includes are not sorted.
2. Are they all needed?

Reply:
Removed unnecessary imports.